src/Codegen/Constraints/UntypedBuilder.php

```diff
@@ -242,7 +242,7 @@ private function generateAllOfChecks(vec<TSchema> $schemas, HackBuilder $hb): vo
   private function generateAnyOfChecks(vec<TSchema> $schemas, HackBuilder $hb): void {
     $schema_builders = Vec\map_with_key(
       $schemas,
-      ($index, $schema) ==> $schema_builder =
+      ($index, $schema) ==>
```

Comment:
HHAST complained about the unused variable and I complained about the assignment_as_an_expression.